run.sh

```diff
@@ -293,7 +294,8 @@ workload_4_java() {
                                 -api "${API}" \
                                 -object_size="${OBJECT_SIZE}..${OBJECT_SIZE}" \
                                 -workers="${WORKERS}" \
-                                -samples="${SAMPLES}"
+                                -samples="${SAMPLES}" \
+                                -warmup="60"
```

Comment:
Can we pass this through instead so it can be changed without updating this script?

Reply:
sure, converting to draft while i test and will repost..